src/algokit/core/tasks/mint/models.py

```diff
@@ -74,13 +74,15 @@ def to_file_path(self) -> Path:
             raise ValueError(f"Failed to decode JSON from file {file_path}: {err}") from err
 
     @classmethod
-    def from_json_file(cls: type["TokenMetadata"], file_path: Path | None) -> "TokenMetadata":
+    def from_json_file(cls, file_path: Path | None, name: str, decimals: int | None = 0) -> "TokenMetadata":
```

Comment:
Is it ever possible now for decimals to be None? We can potentially switch this to `decimals: int`?